index.html

```diff
@@ -5,47 +5,50 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="stylesheets/style.css">
   <script src="https://use.fontawesome.com/1fb0eb3889.js"></script>
+  <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Slabo+27px" rel="stylesheet">
   <title>Where work happens | Slack</title>
 </head>
 
 <body>
-  <nav>
+  <nav class="nav-bar">
     <img src="./images/slack-logo.png" alt="slack_logo">
-    <ul>
+    <ul class="menu">
       <li>Why Slack?</li>
       <li>Pricing</li>
       <li>About Us</li>
       <li>Your Workspaces</li>
     </ul>
   </nav>
-  <header>
+  <header class="intro">
     <img src="./images/home_talking.png" alt="home_work_happening">
     <h1>Where Work Happens</h1>
     <p>When your team needs to kick off a project, hire a new employee, deploy some code, review a sales contract, finalize next year's budget, measure an A/B test, plan your next office opening, and more, Slack has you covered.</p>
-    <button type="button" name="button">GET STARTED</button>
-    <p>Already using Slack? <a href="#">Sign in</a></p>
+    <button class="extra start" type="button" name="button">GET STARTED</button>
+    <p class="note">Already using Slack? <a href="#">Sign in</a></p>
   </header>
-  <main>
-    <div>
+  <main class="container">
+    <div class="content">
       <h2>You´re in good company</h2>
       <p>Millions of people around the world have already made Slack the place where their work happens.</p>
-      <button>DISCOVER WHY</button>
-      <img src="./images/airbnb-logo.png" alt="">
-      <img src="./images/capital-one-logo.png" alt="">
-      <img src="./images/harvard-logo.png" alt="">
-      <img src="./images/los-angeles-times-logo.png" alt="">
-      <img src="./images/oracle-logo.png" alt="">
-      <img src="./images/ticketmaster-logo.png" alt="">
+      <button class="extra discover">DISCOVER WHY</button>
+      <img class="partner" src="./images/airbnb-logo.png" alt="">
```

Comment:
Now that you have edited them maybe you could add the content for the alt attribute :)